### DISCUSSION TITLE
Yolov5 on RTX 4080 or 4090

### DISCUSSION THREAD

**Discussion by barney2074** [ultralytics/yolov5#10667] [Q&A]
I'm just wondering if anyone has any benchmark or anecdotal feedback on the use of RTX4080 or 4090 GPU with YoloV5 for both model training and inferencing. The availability/price of these GPUs seems to be improving and it might be time for an upgrade (I'm currently using a 3060)

This opencv article looks promising:
https://learnopencv.com/performance-comparison-of-yolo-models/

thank you

Andrew

**Comment by rd-neosoft** [comment]
Hey, do you have any benchmarks now?
